accounts/pkg/service/v0/index.go

```diff
@@ -63,11 +63,11 @@ func recreateContainers(idx *indexer.Indexer, cfg *config.Config) error {
 	}
 
 	// Groups
-	if err := idx.AddIndex(&proto.Group{}, "OnPremisesSamAccountName", "Id", "groups", "unique", nil, true); err != nil {
+	if err := idx.AddIndex(&proto.Group{}, "OnPremisesSamAccountName", "Id", "groups", "unique", nil, false); err != nil {
 		return err
 	}
 
-	if err := idx.AddIndex(&proto.Group{}, "DisplayName", "Id", "groups", "non_unique", nil, true); err != nil {
+	if err := idx.AddIndex(&proto.Group{}, "DisplayName", "Id", "groups", "non_unique", nil, false); err != nil {
```

Comment:
Wait, why is searching for display names supposed to be case sensitive?

Reply:
Because if you are creating a new group with via the OCS API like this
```
curl -sk -u moss:vista -X POST 'https://localhost:9200/ocs/v2.php/cloud/groups' -d 'groupid=Group1'
```
Then the `OnPremisesSamAccountName` and `DisplayName` are the same. And if there would be the groups `Group1` and `group1` then you couldn't search for them using the `DisplayName` if it was case insensitive.